### PR TITLE
Test/edit profile profile fe

### LINKED ISSUE #22: FE - Profile Testing
Tasks: 
- [x] Write tests to check that Profile & EditProfile are correctly rendering props from state
- [x] Write tests to check that updated inputs/properties are rendering on the Profile page after being saved in EditProfile.

### PR DESCRIPTION
**Pull request by rachael-t**
# PR Description
Please include a summary of the change:
- Writes and passes unit tests for both the EditProfile and Profile component to ensure data is being displayed properly, and buttons are correctly routing a user when clicked.
Screen Shot 2020-07-11 at 11 37 52 AM

Commit Emojis:
:bar_chart: new tests

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
- Testing
- closes #22 

Note any questions or struggles as well.
- Had a difficult time understanding and working in mock functions for the action creators. Commented-out attempted code in case there is time to revisit these tests.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [X] My code follows the Turing style guidelines
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by infamouskeyduster — APPROVED** [review]
@rachael-t Great job getting some unit tests setup! The setup seems quite a bit different that what we are used to. Way to figure this out! I understand that testing with Redux is still an unknown, but I appreciate your solid attempt!

File: code/web/src/modules/user/EditProfile.test.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    return render(
+        <MemoryRouter>
+            <Provider store={store}>
+                <EditProfile />
+            </Provider>
+        </MemoryRouter>
+    )
+}
+
+describe('EditProfile', () => {
```
Comment (zachholcomb) [inline code/web/src/modules/user/EditProfile.test.js:37]: Nice job rachael!

File: code/web/src/modules/user/EditProfile.test.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        expect(userName).toBeInTheDocument();
+        expect(userEmail).toBeInTheDocument();
+        expect(userDescritpion).toBeInTheDocument();
+        expect(userAddress).toBeInTheDocument();
+        expect(userCity).toBeInTheDocument();
+        expect(userState).toBeInTheDocument();
+        expect(userZip).toBeInTheDocument();
+    })
+
+    // Attempt to mock out the functionality for when a user clicks the save button and the action creator is fired to update the state in store - did not get passing
```
Comment (atkinson-daniel) [inline code/web/src/modules/user/EditProfile.test.js:82]: I'm guessing this is a test you all are still working to finish?

**Review by atkinson-daniel — APPROVED** [review]
Looks good!

**Review by zachholcomb — APPROVED** [review]

**Pull request merged**